qiskit_aer/backends/aer_compiler.py

```diff
@@ -672,19 +877,22 @@ def _assemble_op(aer_circ, inst, qubit_indices, clbit_indices, is_conditional, c
     elif name == "diagonal":
         aer_circ.diagonal(qubits, params, conditional_reg, label if label else "diagonal")
     elif name == "unitary":
-        aer_circ.unitary(qubits, params[0], conditional_reg, label if label else "unitary")
+        aer_circ.unitary(qubits, params[0], conditional_reg, aer_cond_expr,
+                         label if label else "unitary")
     elif name == "pauli":
-        aer_circ.gate(name, qubits, [], params, conditional_reg, label if label else name)
+        aer_circ.gate(name, qubits, [], params, conditional_reg, aer_cond_expr,
+                      label if label else name)
     elif name == "initialize":
         _check_no_conditional(name, conditional_reg)
         aer_circ.initialize(qubits, params)
     elif name == "roerror":
         _check_no_conditional(name, conditional_reg)
         aer_circ.roerror(qubits, params)
     elif name == "multiplexer":
-        aer_circ.multiplexer(qubits, params, conditional_reg, label if label else name)
+        aer_circ.multiplexer(qubits, params, conditional_reg, aer_cond_expr,
+                             label if label else name)
```

Comment:
I'm surprised that this passes the lint CI, because this line splitting isn't compatible with `black`, which I thought Aer was using.